build/lib/builder.js

```diff
@@ -17,7 +17,7 @@ const DIST_DIR = path.join(ROOT_DIR, 'dist');
 const TMP_DIR = path.join(DIST_DIR, 'tmp');
 
 // platforms/OS mappings
-const ALL_OSES = [ 'win32', 'linux', 'osx' ];
+const ALL_OSES = [ 'win32', 'osx' ]; // NOTE: 'linux' could be added, but is not officially supported.
```

Comment:
Provides slightly faster packaging time as _mobilesdk-linux.zip_ is not generated.

Reply:
is this just for the CI or will that also stop me from building an SDK for linux at the end?

Reply:
No it shouldn't, however I updated this PR so specifying `--all` will only omit linux on non-linux hosts.